build_msvc/vcpkg.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "bitcoin-core",
+  "version-string": "0.20.99",
```

Comment:
Does this need to be bumped at the same time the other version string is bumped?

Reply:
"End users can set these fields to whatever they want, but the name must be all lowercase characters." From the Link in OP


So I'd suggest to set this to something not entangled with the bitcoin core version. Maybe just `"1"`?

Reply:
You beat me to it and as you've mentioned it's arbitrary. The version field (https://vcpkg.readthedocs.io/en/latest/specifications/manifests/) seems to be intended for use when creating vcpkg port files rather than consuming them. It is a mandatory field though so has to be set to something. I've gone ahead and set it to `1` to avoid any perception it relates to any other release or version numbers.